build/flaws.js

```diff
@@ -304,6 +304,17 @@ function injectBrokenLinksFlaws(doc, $, { rawContent }, level) {
     // that function knows which match it's referring to.
     checked.set(href, checked.has(href) ? checked.get(href) + 1 : 0);
 
+    // Note, a lot of links are like this:
+    //  <a href="/docs/Learn/Front-end_web_developer">
+    // which means the author wanted the link to work in any language.
+    // When checking it against disk, we'll have to assume a locale.
+    const hrefSplit = href.split("#");
+    let hrefNormalized = hrefSplit[0];
+    if (hrefNormalized.startsWith("/docs/")) {
+      const thisDocumentLocale = doc.mdn_url.split("/")[1];
+      hrefNormalized = `/${thisDocumentLocale}${hrefNormalized}`;
+    }
```

Comment:
All of these lines are not new. They're just lifted from inside the `} else if  (..) {` block later. 
It's because now another one of those `} else if (...) {` needs the same `hrefNormalized` variable